howso/direct/client.py

```diff
@@ -1213,9 +1263,19 @@ def copy_trainee(
                 'The copy trainee parameter `resources` is deprecated and will be removed in '
                 'a future release. Please use `runtime` instead.', DeprecationWarning)
 
+        transactional = runtime is not None and runtime.get('transactional', False)
```

Comment:
Should take into account the persistence level as well

Reply:
Changed.  Both the persistence and transactional settings need to take into account the original trainee's settings, if they aren't passed into this API call; the code as you've highlighted this here also turns off transactional mode if it wasn't in the new-trainee options.